websites/N/Night.design/presence.ts

```diff
@@ -0,0 +1,86 @@
+const presence = new Presence({
+    clientId: "1031730262092042321"
+})
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo",
+	}
+
+    presenceData.startTimestamp = Math.floor(Date.now() / 1000);
+    presenceData.buttons = [
+        {
+            label: "View Website",
+            url: "https://night.design/"
+        },
+        {
+            label: "Actually Page",
+            url: document.location.href
+        }
+    ]
```

Comment:
Static pages are not allowed

Reply:
I have remove it.

Reply:
> I have remove it.

In addition to the "actually page", [buttons to main pages are prohibited](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets).